### PR TITLE
Add mastodon link

### PR DESCRIPTION
**Pull request by michaelosthege**

### CODE REVIEW

**Review by twiecki — APPROVED** [review]

**Comment by twiecki** [comment]
Can you also add LinkedIn: https://www.linkedin.com/company/pymc

**Comment by reshamas** [comment]
Can we add in YouTube too?
This is our list of social media platforms

- Discourse:  https://discourse.pymc.io/
- Twitter:  https://twitter.com/pymc_devs
- YouTube:  https://www.youtube.com/c/PyMCDevelopers 
- LinkedIn: https://www.linkedin.com/company/pymc/ 
- Mastodon:  https://bayes.club/@pymc

File: README.rst [hunk 1]
```diff
@@ -80,7 +80,8 @@ Please choose from the following:
 Contact
 =======
 
-We are using `discourse.pymc.io <https://discourse.pymc.io/>`__ as our main communication channel. You can also follow us on `Twitter @pymc_devs <https://twitter.com/pymc_devs>`__ for updates and other announcements.
+We are using `discourse.pymc.io <https://discourse.pymc.io/>`__ as our main communication channel.
+You can also follow us on `Mastodon @pymc@bayes.club <https://bayes.club/@pymc>`__ and `Twitter @pymc_devs <https://twitter.com/pymc_devs>`__ for updates and other announcements.
```
Comment (reshamas) [inline README.rst]:
```suggestion
You can also follow us on these social media platforms for updates and other announcements:    
- `LinkedIn @pymc <https://www.linkedin.com/company/pymc/>`__ 
- `YouTube @PyMCDevelopers <https://www.youtube.com/c/PyMCDevelopers>`__
- `Twitter @pymc_devs <https://twitter.com/pymc_devs>`__ 
- `Mastodon @pymc@bayes.club <https://bayes.club/@pymc>`__ 
```

File: README.rst [hunk 2]
```diff
@@ -81,11 +81,11 @@ Contact
...
 We are using `discourse.pymc.io <https://discourse.pymc.io/>`__ as our main communication channel.
-You can also follow us on these social media platforms for updates and other announcements:    
-- `LinkedIn @pymc <https://www.linkedin.com/company/pymc/>`__ 
+You can also follow us on these social media platforms for updates and other announcements:
+- `LinkedIn @pymc <https://www.linkedin.com/company/pymc/>`__
 - `YouTube @PyMCDevelopers <https://www.youtube.com/c/PyMCDevelopers>`__
-- `Twitter @pymc_devs <https://twitter.com/pymc_devs>`__ 
-- `Mastodon @pymc@bayes.club <https://bayes.club/@pymc>`__ 
+- `Twitter @pymc_devs <https://twitter.com/pymc_devs>`__
+- `Mastodon @pymc@bayes.club <https://bayes.club/@pymc>`__
```
Comment (reshamas) [inline README.rst]: @michaelosthege 
Perhaps we should move this list to *after* the text below

File: README.rst [hunk 3]
```diff
@@ -81,11 +81,11 @@ Contact
...
 
 To ask a question regarding modeling or usage of PyMC we encourage posting to our Discourse forum under the `“Questions” Category <https://discourse.pymc.io/c/questions>`__. You can also suggest feature in the `“Development” Category <https://discourse.pymc.io/c/development>`__.
```
Comment (reshamas) [inline README.rst:91]:
```suggestion
To ask a question regarding modeling or usage of PyMC we encourage posting to our Discourse forum under the `“Questions” Category <https://discourse.pymc.io/c/questions>`__. You can also suggest feature in the `“Development” Category <https://discourse.pymc.io/c/development>`__.

You can also follow us on these social media platforms for updates and other announcements:
- `LinkedIn @pymc <https://www.linkedin.com/company/pymc/>`__
- `YouTube @PyMCDevelopers <https://www.youtube.com/c/PyMCDevelopers>`__
- `Twitter @pymc_devs <https://twitter.com/pymc_devs>`__
- `Mastodon @pymc@bayes.club <https://bayes.club/@pymc>`__
```

**Comment by michaelosthege** [comment]
FYI, we forgot line breaks before and after the enumeration..

I fixed this with a force-push to main right after I merged and saw that it was broken..

(We should move from RST to Markdown 🤐)

**Comment by twiecki** [comment]
> I fixed this with a force-push to main

You're a mad man! 😆

**Pull request merged**